docs/team/sp4ce-cowboy.md

```diff
@@ -3,156 +3,164 @@ layout: page
-  [studies](https://www.sciencedirect.com/science/article/abs/pii/S0167811623000599])
-  have shown a positive correlation between color complexity and user engagement.
-  Additionally, [this](https://www.diva-portal.org/smash/record.jsf?pid=diva2%3A1788177&dswid=-2283) study also
-found out that certain design elements contribute to increase user engagement by young adults.
-  Given that our target users (university students) are mostly young adults, The UI components are intended to
-increase target user engagement with UniCa$h, which would lead to overall improvements in their personal
-financial management. 
-
-
-#### Decommissioned Features
-
-Below are some of the features that I had implemented prior, but
-were removed due to certain considerations.
-
-* **Fuzzy-Find Transactions**: *feature yet to be implemented*
-* **Filter Transactions**: *feature yet to be implemented*
-* **Local User Guide**: *feature yet to be implemented*
-
-
-#### Code
-* Code contributed: [RepoSense Link](https://nus-cs2103-ay2324s1.github.io/tp-dashboard/?search=T16-3&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code&since=2023-09-22&tabOpen=true&tabType=authorship&tabAuthor=sp4ce-cowboy&tabRepo=AY2324S1-CS2103-T16-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
-
-#### Documentation
+  * The `CommandBox` can remember up to a maximum of 10 previous user inputs,
+  which can be traversed with the up/down arrow keys. `ESC` will clear the command box.
+  **Justification:** To provide our CLI-proficient target users a more intuitive,
+  convenient and efficient means for them to interact with UniCa$h, similar to a real CLI.
+
+* **UI/UX Visual Features** 
+  * A **StyleSheet class** [PRs: [#96](https://github.com/AY2324S1-CS2103-T16-3/tp/pull/96)] was created such that all components involving any form of graphical
+  representation have a centralized source of reference. **[Highlights/Possible Future Addition]:**
+  The universality and extensibility of the StyleSheet class is such that it is possible to allow the
+  user themselves to create and select certain custom themes which can then be applied across
+  all visual elements in UniCa$h.
+  * **Hash-based Category Coloring** [PRs: [#96](https://github.com/AY2324S1-CS2103-T16-3/tp/pull/96)]
+  This feature allows for a consistent color scheme across unique categories without having to hard-code colors, as 
+  there is no limit for total category count in UniCa$h.
+  **Justification:** To provide a seamless and consistent visual experience across UniCa$h for a better user-experience.
+  _**Highlights:** The motivation for color consistency comes from these studies that show positive correlation between
```

Comment:
Minor nit but maybe quick link the "these links" with the actual studies link

Reply:
Oh my bad, thanks for catching that, I forgot to add the links!

Reply:
I've added the links, thanks again